pkg/cloudprovider/suite_test.go

```diff
@@ -215,10 +218,14 @@ var _ = Describe("CloudProvider", func() {
 	})
 	Context("Node Drift", func() {
 		var validAMI string
+		var validSecurityGroup string
 		var selectedInstanceType *corecloudproivder.InstanceType
 		var instance *ec2.Instance
+		var nodeTemplateController corecontroller.Controller
 		BeforeEach(func() {
 			validAMI = fake.ImageID()
+			validSecurityGroup = fake.SecurityGroupID()
+			nodeTemplateController = nodetemplate.NewController(env.Client, awsEnv.SubnetProvider, awsEnv.SecurityGroupProvider, awsEnv.AMIProvider)
```

Comment:
We probably don't need this controller and a fake API call, we can probably just set the status to what we want it to be, since this is really about testing the cloud provider functions, not the node template controller.